apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -147,18 +151,24 @@ export default function BaseMegaMenu() {
     close();
   });
 
+  const search = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    // eslint-disable-next-line no-alert
+    alert(`Successfully found 10 results for ${inputValue}`);
+  };
+
   return (
     <div className="w-full h-full">
       {isOpen && <div className="fixed inset-0 bg-neutral-500 bg-opacity-50 transition-opacity" />}
       <header
         ref={menuRef}
-        className="flex justify-center w-full border-0 bg-primary-700 border-neutral-200 h-14 md:relative md:h-20 md:z-10"
+        className="flex justify-center w-full py-2 md:py-5 border-0 bg-primary-700 border-neutral-200 md:relative md:z-10"
       >
-        <div className="flex items-center flex-nowrap justify-start h-full max-w-[1536px] w-full px-4 md:px-10">
+        <div className="flex items-center flex-wrap md:flex-nowrap justify-start h-full max-w-[1536px] w-full px-4 md:px-10">
           <a
             href="#"
             aria-label="SF Homepage"
-            className="flex shrink-0 mr-2 text-white md:mr-10 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
+            className="flex shrink-0 mr-2 md:order-first text-white md:mr-10 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
```

Comment:
When using `order` we may get unexpected results when tabbing through interactive elements. Like in this case, on mobile devices it looks like that. I'm not sure if that's ok.

https://github.com/vuestorefront/storefront-ui/assets/34419118/01d18042-114a-4150-95c8-58ca24ab09a7